low-level/framework/messaging/ingress_processor.py

```diff
@@ -108,10 +108,17 @@ def initialize(self, conf_reader, msgQlist, product):
 
         self._read_config()
         producer_initialized.wait()
-        self._consumer = MessageConsumer(consumer_id=self._consumer_id,
-                                         consumer_group=self._consumer_group,
-                                         message_types=[self._message_type],
-                                         auto_ack=False, offset=self._offset)
+        self.create_MsgConsumer_obj()
+
+    def create_MsgConsumer_obj(self):
+        self._consumer = None
+        try:
+            self._consumer = MessageConsumer(consumer_id=self._consumer_id,
+                consumer_group=self._consumer_group,
+                message_types=[self._message_type],
+                auto_ack=False, offset=self._offset)
+        except Exception as err:
+            logger.error('Instance creation for MessageConsumer class failed due to %s' % err)
```

Comment:
Ideally, we should avoid generic exceptions. Please check if you can put specific exception here, otherwise good to go.

Reply:
Here we got error- 'KafkaError{code=_ALL_BROKERS_DOWN,val=-187,str="1/1 brokers are down"}' which is not getting handled by MessageBusError exception, hence added generic exception.

Reply:
Okay.